### DISCUSSION TITLE
Copying

### DISCUSSION THREAD

**Discussion by ranrhos** [HarshPatel5940/Discord-Helper-Bot#10] [General]
I'm gonna help you out, and add more features + commands, and I will reply back here with the updated version; you may / feel free to check it out at a later date, although I will be inactive from,

17th - 23rd June, because Exams.

**Comment by HarshPatel5940** [comment]
Oh k Great! Even I have exams from May 7 (2d ago) - June 13. + Other exam on June 20 probably

I will be looking forward to talk to you when we can : )


Excited to add more and more things to the bot ✌️

**Reply by ranrhos** [reply]
Hi, thank you for the reply back. I would like to ask if you have a discord so we can speak more than on here?

Best regards,
Tribe

**Reply by HarshPatel5940** [reply]
Yes you can contact me on [HarshPatel5940#3210](https://discord.com/users/448740493468106753)
